query-node/mappings/src/council.ts

```diff
@@ -479,13 +497,16 @@ export async function council_NewCouncilElected({ event, store }: EventContext &
     electedAtBlock: event.blockNumber,
     electedAtTime: new Date(event.blockTimestamp),
     electedAtNetwork: CURRENT_NETWORK,
-    councilElections: oldElectedCouncil?.nextCouncilElections || [],
-    nextCouncilElections: [],
+    councilElections: oldElectedCouncil?.nextCouncilElections || [], // always empty ?
+    nextCouncilElections: [], // always empty ?
```

Comment:
Those fields are derived, so I think it's best to leave them empty.
Also `oldElectedCouncil?.nextCouncilElections` should always be empty at this point, so it really makes no difference.
```suggestion
    councilElections: [],
    nextCouncilElections: [],
```

Reply:
Thanks for clarifying this. Yes I did notice they were derived in the schema definitions, and I did ask why we set them at all if they are indeed derived https://github.com/Joystream/joystream/issues/3573#issuecomment-1143211675